tests/unit/store/unload-test.js

```diff
@@ -86,7 +86,7 @@ test("unload a record", function(assert) {
         store.unloadRecord(record);
       });
 
-      assert.equal(get(record, 'hasDirtyAttributes'), false, "record is not dirty");
```

Comment:
Hmm, not sure what the expected behavior of `hasDirtyAttributes` is here. I will bring this up in the next meeting. Good point @ryanpatrickcook 👍